JfkWebApiSkills/JfkWebApiSkills/HocrGenerator/HocrDocument.cs

```diff
@@ -41,7 +41,7 @@ public class HocrPage
         public HocrPage(OcrImageMetadata imageMetadata, int pageNumber, Dictionary<string,string> wordAnnotations = null)
         {
             // page
-            metadata.WriteLine($"<div class='ocr_page' id='page_{pageNumber}' title='image \"{imageMetadata.ImageStoreUri}\"; bbox 0 0 {imageMetadata.Width} {imageMetadata.Height}; ppageno {pageNumber}'>");
+            metadata.WriteLine($"<div class='ocr_page' id='page_{pageNumber}' data-bbox='image \"{imageMetadata.ImageStoreUri}\"; bbox 0 0 {imageMetadata.Width} {imageMetadata.Height}; ppageno {pageNumber}'>");
```

Comment:
This has more than just bbox data, also has the blob URI and page number, and some other bits of metadata for lines as well (baseline, x_size, etc.) You may want to choose a more generic name.

Reply:
Good point...also looks like it breaks some of the other react code that is pulling from that field that I hadn't accounted for. Will need to look at it further.